src/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>my-project</title>
+</head>
+<body>
+Hello, world!
+</body>
```

Comment:
Remember about proper indentation:
```suggestion
	  <body>
	  		Hello, world!
	  </body>
```

Reply:
исправил